source/functions/BEMIO/readAQWA.m

```diff
@@ -35,7 +35,7 @@
 
 hydro(F).code   = 'AQWA';
 V182            = 0; % Set AqwaVersion flag "Version 18.2 or larger" to 0
-tmp             = strsplit(ah1Filename,{' ','\','.'});
+tmp             = strsplit(ah1Filename,{' ','\', '/', '.'});
```

Comment:
Update using MATLAB ``filesep`` to be most robust:

    tmp             = strsplit(ah1Filename,{' ', filesep, '.'});